client/tx/factory.go

```diff
@@ -189,6 +190,12 @@ func (f Factory) WithKeybase(keybase keyring.Keyring) Factory {
 	return f
 }
 
+// WithFromName returns a copy of the Factory with updated fromName
```

Comment:
Can get this more descriptive and explain that it should most likely be clientCtx.FromName?

Reply:
Okay